articlequality/feature_lists/fawiki.py

```diff
@@ -26,13 +26,29 @@
     "Who|چه کسی|چه‌کسی", name="fawiki.revision.who_templates")
 main_article_templates = wikitext.revision.template_names_matching(
     "Main|اصلی", name="fawiki.main_article_templates")
+CITE_TEMPLATES = [
+    r"Cite",
+    r"Harvard[_ ]citation[_ ]no[_ ]brackets", r"harvnb",
+    r"Harvard citation", r"harv",
+    r"Harvard citation text", r"harvtxt",
+    r"Harvcoltxt",
+    r"Harvcol",
+    r"Harvcolnb",
+    r"Harvard citations", r"harvs",
+    r"Harvp",
+    r"یادکرد"
+]
 cite_templates = wikitext.revision.template_names_matching(
-    r"cite|یادکرد", name="fawiki.revision.cite_templates")
+    "|".join(CITE_TEMPLATES), name="fawiki.revision.cite_templates")
+shortened_footnote_templates = wikitext.revision.template_names_matching(
+    "sfn", name="fawiki.revision.shortened_footnote_templates")
```

Comment:
Question: Is there a reason why you didn't include stylistic alternatives/aliases to sfn, such as `{{Shortened footnote template}}`, `{{Sfnp}}`, `{{Sfnm}}`? They are listed in the [enwiki features](https://github.com/wikimedia/articlequality/blob/master/articlequality/feature_lists/enwiki.py#L43) and also you can find them in the fawiki doc for the [sfn template](https://fa.wikipedia.org/wiki/%D8%A7%D9%84%DA%AF%D9%88:Sfn#Author-date_citation_templates).

Reply:
Thanks for the review, Aiko. Earlier on, I was following [ptwiki's implimentation](https://github.com/wikimedia/articlequality/blob/master/articlequality/feature_lists/ptwiki.py#L54-L55) but I've now included sfn aliases similar to [enwiki's implemention](https://github.com/wikimedia/articlequality/blob/master/articlequality/feature_lists/enwiki.py#L43-L51).